service/storage_service.cc

```diff
@@ -284,7 +284,7 @@ void storage_service::prepare_to_join(
         if (db::system_keyspace::bootstrap_complete()) {
             throw std::runtime_error("Cannot replace address with a node that is already bootstrapped");
         }
-        std::tie(_bootstrap_tokens, _cdc_streams_ts) = prepare_replacement_info(initial_contact_nodes, loaded_peer_features, do_bind).get0();
+        std::tie(_bootstrap_tokens) = prepare_replacement_info(initial_contact_nodes, loaded_peer_features, do_bind).get0();
```

Comment:
Why go with a single-element tuple? Is this info likely to be extended soon?

Reply:
Unlikely. There's no reason it's a tuple at this moment besides me being too lazy to remove it. But I can do that.